compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/bufferize_to_allocation_pack.mlir

```diff
@@ -0,0 +1,143 @@
+// RUN: iree-opt --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(iree-amdaie-bufferize-to-allocation{memory-space=1})))' --split-input-file %s | FileCheck %s --check-prefix=CHECK-1
+// RUN: iree-opt --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(iree-amdaie-bufferize-to-allocation{memory-space=2})))' --split-input-file %s | FileCheck %s --check-prefix=CHECK-2
+
+#executable_target_elf = #hal.executable.target<"amd-aie", "elf", {target_arch = "chip-tbd"}>
```

Comment:
I am fixing this in a separate PR, but we dont need the whole `hal` boiler plate. You can make this pass work on func.func and just write a simple function test.